src/game_api/game_manager.hpp

```diff
@@ -101,14 +105,14 @@ struct RawInput
 struct InputDevice
 {
     // No idea what these actually do, better not to expose this anyway

```

Comment:
*continues to expose it*

Reply:
Well yeah technically I exposed the type, but removed the only place where it was used in.